apps/frontend/src/app/5_pages/BitocracyPage/components/LiveProposals/LiveProposals.tsx

```diff
@@ -1,13 +1,67 @@
-import React, { FC } from 'react';
+import React, { FC, useCallback, useEffect, useState } from 'react';
 
 import { t } from 'i18next';
+import { useNavigate } from 'react-router-dom';
 
+import { Paragraph, Table } from '@sovryn/ui';
+
+import { useBlockNumber } from '../../../../../hooks/useBlockNumber';
 import { translations } from '../../../../../locales/i18n';
+import { Proposal } from '../../../../../utils/graphql/rsk/generated';
+import { generateRowTitle } from '../../BitocracyPage.utils';
+import { useGetProposals } from '../../hooks/useGetProposals';
+import { ProposalCardsMobile } from '../ProposalCardsMobile/ProposalCardsMobile';
+import { columnsConfig } from './LiveProposals.constants';
+
+const LiveProposals: FC = () => {
+  const navigate = useNavigate();
+  const { loading, data: proposals } = useGetProposals();
+  const { value: blockNumber } = useBlockNumber();
+  const [activeProposals, setActiveProposals] = useState<Proposal[]>([]);
+
+  const handleRowClick = useCallback(
+    (proposal: Proposal) => navigate(`/bitocracy/${proposal.proposalId}`),
```

Comment:
```suggestion
    (proposal: Proposal) => navigate(`/bitocracy/${proposal.id}`),
```
use `id`, `proposalId` is not unique